### DISCUSSION TITLE
How to test performance

### DISCUSSION THREAD

**Discussion by zhanghenan-dev** [microsoft/msquic#2216] [Q&A] [answered]
How to test msquic performance? Is it performance.ps1? What are the parameters ? Can msquic start a client and server to test performance ？

I just used cmake build msquic, and test quicsample(server and client).
Now I want to test msquic performance on my centos, but didn't know how to test it.

**Comment by nibanks** [comment, marked as answer]
I've created #2217 to track adding documentation for this. You should be using `secnetperf` (get's built by default). On the server side, simply run without args. On the client side do `secnetperf -test:tput -?` or `secnetperf -test:rps -?` for additional help text.

**Reply by zhanghenan-dev** [reply]
Thanks! I got it
